rubocop.yml

```diff
@@ -21,12 +22,14 @@ Metrics/CyclomaticComplexity:
   Max: 9
 
 Metrics/ClassLength:
+  Enabled: false
   Max: 115
 
 Metrics/LineLength:
   Enabled: false
 
 Metrics/MethodLength:
+  Enabled: false
```

Comment:
Could we use the `--fail-level warning` flag instead of disabling the checks? (Happy to leave these disabled here if we are to include them on Sonarqube)